src/rules/pylint/rules/too_many_args.rs

```diff
@@ -0,0 +1,18 @@
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::registry::Diagnostic;
+use crate::violations;
+use rustpython_ast::{Arguments, Stmt};
+
+/// PLR0913
+pub fn too_many_args(checker: &mut Checker, args: &Arguments, stmt: &Stmt) {
+    if args.args.len() > checker.settings.pylint.max_args {
```

Comment:
I suspect that this also needs to include `kwargs`, `kwonly_args`, and the other fields on `Arguments`. Can you check?

Reply:
This rule only checks for args: https://github.com/PyCQA/pylint/blob/af555eddf6ecdb1993345dce728a6b4c38dd4a02/pylint/checkers/design_analysis.py#L519-L524

Reply:
Ohh, interesting! Thanks.